lib/matplotlib/offsetbox.py

```diff
@@ -1424,6 +1424,9 @@ def __init__(self, offsetbox, xy,
         if bboxprops:
             self.patch.set(**bboxprops)
 
+        if len(kwargs):
+            self.update(kwargs)
```

Comment:
No need for the `if`; everything I found didn't use it.
```suggestion
        self.update(kwargs)
```